docs/docs/reference-react-dom.md

```diff
@@ -0,0 +1,70 @@
+---
+id: reference-react-dom
+title: ReactDOM
+permalink: docs/reference-react-dom.html
+next: reference-react-dom-server.html
+redirect_from: "/docs/reference.html"
+---
+
+The `react-dom` package provides DOM-specific methods that can be used at the top level of your app and as an escape hatch to get outside of the React model if you need to. Most of your components should not need to use this module.
+
+ - [`render()`](#render)
+ - [`unmountComponentAtNode()`](#unmountComponentAtNode)
+ - [`findDOMNode()`](#findDOMNode)
+
+## Reference
+
+### `render()`
+
+```javascript
+ReactDOM.render(
+  element,
+  container,
+  [callback]
+)
+```
+
+Render a ReactElement into the DOM in the supplied `container` and return a [reference](/react/docs/more-about-refs.html) to the component (or returns `null` for [stateless components](/react/docs/reusable-components.html#stateless-functions)).
+
+If the ReactElement was previously rendered into `container`, this will perform an update on it and only mutate the DOM as necessary to reflect the latest React component.
+
+If the optional callback is provided, it will be executed after the component is rendered or updated.
+
+> Note:
+>
+> `ReactDOM.render()` controls the contents of the container node you pass in. Any existing DOM elements
+> inside are replaced when first called. Later calls use React’s DOM diffing algorithm for efficient
+> updates.
+>
+> `ReactDOM.render()` does not modify the container node (only modifies the children of the container). In
+> the future, it may be possible to insert a component to an existing DOM node without overwriting
+> the existing children.
+>
+> `ReactDOM.render()` currently returns a reference to the root `ReactComponent` instance. However, using this return value is legacy
```

Comment:
Do we need this `ReactComponent` pascal case notation here either? I would rather see us consistently use "component instance" instead. `ReactComponent` sounds like some sort of library-provided class which it is not.